Rephrase section of smart contracts developers docs (#6539)
ethereum/ethereum-org-website: src/content/developers/docs/smart-contracts/index.md

This logic is programmed into the vending machine.

> A smart contract, like a vending machine, has logic programmed into it. Here's a simple example of how this vending machine might look as a smart contract:

Reviewer: ```suggestion
A smart contract, like a vending machine, has logic programmed into it. Here's a simple example of how this vending machine would look if it was a smart contract:
```
